treee111/wahooMapsCreator
# [FEAT] Add shop/bicycle POIs, show also "ways" POIs

## This PR…

- adds "ways"-POIs to the output together with the already existing "pois"-POIs of the same "type" to the map, e.g. `amenity/cafe` (via `tag-wahoo-poi.xml`)
- adds bicycle shops (`shop/bicycle`) to the map 
- do some adjustments to railway POIs
- closes #144
- closes #159

## How to test

1. do normal processing for your area
2. check if bikeshops appear
3. check for railway stations / stops

## Pull Request Checklist
    + Especially the [Pull-Requests](https://github.com/treee111/wahooMapsCreator/blob/develop/.github/CONTRIBUTING.md#Pull-Requests) section
